generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -378,6 +378,11 @@ def cfg_samp_clk_timing(
                 active_edge_raw=active_edge, sample_mode_raw=sample_mode,
                 samps_per_chan=samps_per_chan))
 
+    def cfg_time_start_trig(self, task, when, timescale):
+        response = self._invoke(
+            self._client.CfgTimeStartTrig,
+            grpc_types.CfgTimeStartTrigRequest(task=task, when=when, timescale_raw=timescale))
```

Comment:
what type does the grpc lib use for when?

Reply:
google.protobuf.Timestamp
https://github.com/ni/nidaqmx-python/blob/master/src/codegen/protos/nidaqmx.proto#L4690
https://github.com/ni/nidaqmx-python/blob/master/src/handwritten/_grpc_time.py